lib/recaptcha.rb

```diff
@@ -65,10 +65,12 @@ def self.verify_via_api_call(response, options)
     verify_hash['remoteip'] = options[:remote_ip] if options.key?(:remote_ip)
 
     reply = api_verification(verify_hash, timeout: options[:timeout])
-    reply['success'].to_s == 'true' &&
+    success = reply['success'].to_s == 'true' &&
       hostname_valid?(reply['hostname'], options[:hostname]) &&
       action_valid?(reply['action'], options[:action]) &&
       score_above_threshold?(reply['score'], options[:minimum_score])
+
+    return success, reply
```

Comment:
how about not breaking anyones existing code ? :)
```suggestion
    if options[:reply]
      [success, reply]
    else
      success
    end
```

Reply:
Developer's wisdom :)

Reply:
Nice thanks. How about something a little more explicit like `include_reply` or `with_reply` ?

Reply:
either is fine 👍

Reply:
Fab, if the rest is ok with you I'll work on adding tests and doc tomorrow

Reply:
yeah, just needs test + docs :)